proto/axelarnet/v1beta1/tx.proto

```diff
@@ -29,11 +29,14 @@ message ConfirmDepositRequest {
   bytes sender = 1 [ (gogoproto.casttype) =
                          "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
   bytes tx_id = 2 [ (gogoproto.customname) = "TxID" ];
-  cosmos.base.v1beta1.Coin token = 3 [ (gogoproto.nullable) = false ];
+  cosmos.base.v1beta1.Coin token = 3
+      [ (gogoproto.nullable) = false, deprecated = true ];
```

Comment:
Since we're doing an upgrade, can we just remove this field or does that break something?

Reply:
@sammy1991106 thoughts?

Reply:
It doesn't break anything currently and it has nothing to do with how we upgrade. We are launching mainnet now and in the future people will build their own systems creating axelar transactions and doing the signing. I'd like to start getting us used to not breaking transaction encoding every time instead of letting it happen all of a sudden. At one point, we may also launch our own js client library similar to ethers.js and web3.js, and it's good to start practicing how to keep this kind of things non-breaking even between breaking versions. You can't expect all these toolings to be upgraded the moment our chain is upgraded.